### ISSUE TITLE
Unable to resize "Settings" window

### ISSUE THREAD

**Issue by CabeLovlie** [ohtu-intellij/tmc-intellij#103]
This is blocking me from completing the sign-in process.  I cannot resize the window, and many of the settings are outside of the window boarder.

**Comment by cxcorp** [comment]
Could you perhaps upload a screenshot of the window?

**Comment by 1337tester** [comment]
![image](https://cloud.githubusercontent.com/assets/6553766/20861230/e1e50ebc-b98a-11e6-8264-493248ee049a.png)

could be bound to a high resulution DPI display on my notebook

**Comment by CabeLovlie** [comment]
That's exactly how it appeared on my laptop. I've since changed over to a
FHD display though.

**Comment by cxcorp** [comment]
Yeah, definitely looks like an DPI-caused issue. I'll take a look at it.

**Comment by cxcorp** [comment]
@1337tester What scaling percentage are you using currently? What desktop resolution are you using? The scaling percentage can be found from Display Settings and the resolution from the "Advanced display settings" view.

![image](https://cloud.githubusercontent.com/assets/21111572/21117989/ab3df11c-c0c4-11e6-92ee-294dd018defb.png)

**Comment by 1337tester** [comment]
![image](https://cloud.githubusercontent.com/assets/6553766/21118779/ae980848-c0bf-11e6-82e9-bfc0d720f44e.png)

225%,

It is so high because it helps make other application look normal on a laptop screen with resolution 
3840 x 2160

**Comment by 1337tester** [comment]
Quite it relates to this resizing, when I put this setting back to 100% it shows the window correctly:
![image](https://cloud.githubusercontent.com/assets/6553766/21119422/cfba66a8-c0c2-11e6-867a-782c2b7a219c.png)
However as you see it makes everything else unusable for people without eagle-vision
